src/Utils/Util.ts

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Can we rename this file to `sortCustomContextData`? This utility function is very specific to custom context data.